### PR TITLE
Deprecate `though`, add `through` annotation stubs

### LINKED ISSUE #10: Annotation name
Shouldn't annotation `though` be changed to `through` (i.e. "using", "by means of")?

### PR DESCRIPTION
**Pull request by rauanmayemir**
I had to drop required constraint on `though`, otherwise I couldn't find a way to make this a smooth transition.
Upon removal of `though` after one cycle (pun intended), we can switch the required constraint back on for `through`.

Closes #10.

Cc: @roxblnfk

### CODE REVIEW

File: src/Entities.php [hunk 1]
```diff
@@ -132,10 +132,63 @@ protected function normalizeNames(Registry $registry): Registry
...
+                        }
+                    }
+
+                    /**
+                     * We wanted to include composite keys support because it would be harder
+                     * to change the annotation signature later, but cycle itself doesn't support
+                     * composite keys on the ORM schema level, so we include a temporary hack
+                     * that limits the THROUGH inner and outer keys to a single element array.
+                     *
+                     * @see https://github.com/cycle/schema-builder/issues/25
```
Comment (roxblnfk) [inline src/Entities.php:160]: I think the `annotated` package should not have this restriction.

**Review by roxblnfk — COMMENTED** [review]
Please allow code pushing in this branch

File: src/Entities.php (hunk 1 above)
Comment (rauanmayemir) [reply src/Entities.php:160]: Why not? Sometimes, it can get confusing whether the issue is with annotated schemas or data structure in general.

Comment (rauanmayemir) [reply src/Entities.php:160]: The restriction is temporary and is supposed to be lifted once composite keys support is released.

**Review by roxblnfk — APPROVED** [review]

**Pull request merged**